diff --git a/node_modules/@cosmjs/amino/build/pubkeys.js b/node_modules/@cosmjs/amino/build/pubkeys.js
index e9844ef..12f735c 100644
--- a/node_modules/@cosmjs/amino/build/pubkeys.js
+++ b/node_modules/@cosmjs/amino/build/pubkeys.js
@@ -1,30 +1,40 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.isMultisigThresholdPubkey = exports.isSinglePubkey = exports.pubkeyType = exports.isSecp256k1Pubkey = exports.isEd25519Pubkey = void 0;
+'use strict';
+Object.defineProperty(exports, '__esModule', { value: true });
+exports.isMultisigThresholdPubkey =
+  exports.isSinglePubkey =
+  exports.pubkeyType =
+  exports.isSecp256k1Pubkey =
+  exports.isEd25519Pubkey =
+    void 0;
 function isEd25519Pubkey(pubkey) {
-    return pubkey.type === "tendermint/PubKeyEd25519";
+  return pubkey.type === 'tendermint/PubKeyEd25519';
 }
 exports.isEd25519Pubkey = isEd25519Pubkey;
 function isSecp256k1Pubkey(pubkey) {
-    return pubkey.type === "tendermint/PubKeySecp256k1";
+  return pubkey.type === 'tendermint/PubKeySecp256k1';
 }
 exports.isSecp256k1Pubkey = isSecp256k1Pubkey;
+function isBn254Pubkey(pubkey) {
+  return pubkey.type === 'tendermint/PubKeyBn254';
+}
+exports.isBn254Pubkey = isBn254Pubkey;
 exports.pubkeyType = {
-    /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/ed25519/ed25519.go#L22 */
-    secp256k1: "tendermint/PubKeySecp256k1",
-    /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/secp256k1/secp256k1.go#L23 */
-    ed25519: "tendermint/PubKeyEd25519",
-    /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/sr25519/codec.go#L12 */
-    sr25519: "tendermint/PubKeySr25519",
-    multisigThreshold: "tendermint/PubKeyMultisigThreshold",
+  /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/ed25519/ed25519.go#L22 */
+  secp256k1: 'tendermint/PubKeySecp256k1',
+  /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/secp256k1/secp256k1.go#L23 */
+  ed25519: 'tendermint/PubKeyEd25519',
+  /** @see https://github.com/tendermint/tendermint/blob/v0.33.0/crypto/sr25519/codec.go#L12 */
+  sr25519: 'tendermint/PubKeySr25519',
+  bn254: 'tendermint/PubKeyBn254',
+  multisigThreshold: 'tendermint/PubKeyMultisigThreshold',
 };
 function isSinglePubkey(pubkey) {
-    const singPubkeyTypes = [exports.pubkeyType.ed25519, exports.pubkeyType.secp256k1, exports.pubkeyType.sr25519];
-    return singPubkeyTypes.includes(pubkey.type);
+  const singPubkeyTypes = [exports.pubkeyType.ed25519, exports.pubkeyType.secp256k1, exports.pubkeyType.sr25519];
+  return singPubkeyTypes.includes(pubkey.type);
 }
 exports.isSinglePubkey = isSinglePubkey;
 function isMultisigThresholdPubkey(pubkey) {
-    return pubkey.type === "tendermint/PubKeyMultisigThreshold";
+  return pubkey.type === 'tendermint/PubKeyMultisigThreshold';
 }
 exports.isMultisigThresholdPubkey = isMultisigThresholdPubkey;
 //# sourceMappingURL=pubkeys.js.map
